template/en/default/account/email/bounced-emails.html.tmpl

```diff
@@ -0,0 +1,54 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% PROCESS global/header.html.tmpl
+  title = "Bounced Emails"
+%]
+
+<h2>[% title FILTER html %]</h2>
+
```

Comment:
This could do with a bit more explanatory text.

I'd like to see the current bounce reason called out here, an explanation of what that means and what actions we expect users to take to resolve it prior to reactivating delivery (eg. you may have to contact your email provider to ensure there aren't any issues with your mail account).  We should also state here that if your email bounces more than `BOUNCE_REACTIVATION_MAX` times a Bugzilla administrator will need to reactivate your account after performing more checks.